lib/mongoid/interceptable.rb

```diff
@@ -149,6 +149,14 @@ def run_callbacks(kind, with_children: true, skip_if: nil, &block)
     #
     # @api private
     def _mongoid_run_child_callbacks(kind, children: nil, &block)
+      if Mongoid::Config.around_callbacks_for_embeds
```

Comment:
The logic here seems to contradict the documentation for the `around_callbacks_for_embeds` option. Maybe the documentation just has the logic backwards?

Reply:
You are right, thank you. Adjusted the docs.